lib/pdk/util/ruby_version.rb

```diff
@@ -45,10 +45,17 @@ def scan_for_packaged_rubies
         end
 
         def default_ruby_version
+          # For now, the packaged versions will not be using default of 2.4.4.
+          '2.4.4' if PDK::Util.package_install?
+
           # TODO: may not be a safe assumption that highest available version should be default
           versions.keys.sort { |a, b| Gem::Version.new(b) <=> Gem::Version.new(a) }.first
```

Comment:
Should change this line to just call `latest_ruby_version` now to get rid of the duplicated code